src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandler.java

```diff
@@ -80,17 +81,18 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             hearingMessageHelper.sendListAssistCreateHearingMessage(sscsCaseData.getCcdCaseId());
         }
 
+        HearingOptions caseDataHearingOptions = sscsCaseData.getAppeal().getHearingOptions();
+        HearingOptions hearingOptions = caseDataHearingOptions != null ? caseDataHearingOptions : HearingOptions.builder().build();
+
         if (adjournment.getInterpreterRequired() != null) {
-            HearingOptions hearingOptions = HearingOptions.builder().build();
-            if (sscsCaseData.getAppeal().getHearingOptions() != null) {
-                hearingOptions = sscsCaseData.getAppeal().getHearingOptions();
-            }
             DynamicList interpreterLanguage = adjournment.getInterpreterLanguage();
-            hearingOptions.setLanguages(nonNull(interpreterLanguage.getValue()) ? interpreterLanguage.getValue().getLabel() : "");
+            hearingOptions.setLanguages(nonNull(interpreterLanguage.getValue()) ? interpreterLanguage.getValue().getLabel() : NO.getValue());
```

Comment:
Need a test for this
![image](https://user-images.githubusercontent.com/110455092/218980938-bbd5a4b1-e6f5-4640-8dcb-d033e1e617ce.png)